### ISSUE TITLE
TypeError: Cannot read properties of null (reading 'value')

### ISSUE THREAD

**Issue by ai** [kytta/eslint-plugin-node-import#3]
I got an error

```
TypeError: Cannot read properties of null (reading 'value')
Occurred while linting /home/ai/Dev/browsersl.ist/client/data/regions.js:4
Rule: "node-import/prefer-node-protocol"
    at literalVisitor (/home/ai/Dev/browsersl.ist/node_modules/.pnpm/eslint-plugin-node-import@1.0.1_eslint@8.48.0/node_modules/eslint-plugin-node-import/lib/rules/prefer-node-protocol.js:24:22)
```

on the code:

```js
// regions.json is created on `postinstall` step
import { continents, countryCodes } from './regions.json'

export const DEFAULT_REGION = 'alt-ww' // <= here is a problem
```

**Comment by ai** [comment]
Another example from CI

https://github.com/ai/sitnik.ru/actions/runs/6108588738/job/16577896232

**Comment by kytta** [comment]
Should be fixed in #4 and v1.0.2; thanks!
